skimage/filters/thresholding.py

```diff
@@ -578,57 +578,43 @@ def threshold_minimum(image, nbins=256, bias='min', max_iter=10000):
     >>> binary = image > thresh
     """
 
-    def find_local_maxima(hist):
+    def find_local_maxima_idx(hist):
```

Comment:
I have many questions. =P

- Why is this function nested?
- Why not `slope = hist[i + 1] - hist[i]`? You are testing against zero anyway.
- Similarly to your `bias` argument, maybe this function can be removed if we are smoothing the histogram?